Library/Formula/wxmac.rb

```diff
@@ -67,8 +68,13 @@ def install
       "--disable-monolithic"
     ]
 
-    system "./configure", *args
+    system "./configure", "--enable-shared", *args
```

Comment:
@DomT4 Having both static and shared allows users to either link dynamically or statically (not both), depending on their preference. The existing wxmac formula only provides dynamic libraries, but that presents the deployment challenge of bundling all of the dylibs with the executable. I prefer static linking for its simplicity in deployment of my applications.

Before editing the wxmax formula, I took a look at the boost formula for ideas. The boost formula normally provides both dynamic and static libraries, but users can opt out of static libraries with the `--without-static` option. So, I made the wxmac formula behave the same, except defaulting to not include static libraries because that is the original behavior.

However, the boost make system is a bit more powerful than the wxmac make system. Boost allows building and installing both static and dynamic libraries at the same time, but wxmac only allows one choice per `make install`. So, you're right, the second `make install` stage does overwrite the previous install, adding the static libraries. I couldn't find a way around that, without editing the wxmac makefiles or manually installing.